plugins/inputs/kube_inventory/kube_state.go

```diff
@@ -65,6 +70,12 @@ var sampleConfig = `
   ## Overrides resource_exclude if both set.
   # resource_include = [ "deployments", "nodes", "statefulsets" ]
 
+  ## selectors to include and exclude as tags.  Globs accepted.
+  ## Note that an empty array for both will include all selectors as tags
+  ## selector_exclude overrides selector_include if both set.
+  selector_include = []
+  selector_exclude = ["*"]
```

Comment:
Sorry, missed this. Let's initialize these options down in `func init() {` so that upgrades doesn't accidentally get set to include all.  We can then list these commented out here in the sample configuration.

Reply:
done